fixtures/view-transition/src/components/Page.js

```diff
@@ -79,6 +81,23 @@ export default function Page({url, navigate}) {
     // });
   }, [show]);
 
+  const [showModal, setShowModal] = useState(false);
+  const portal = showModal ? (
+    createPortal(
+      <div className="portal">
+        Portal: {!show ? 'A' : 'B'}
```

Comment:
Previously, this would've been thought of as React as being inside the `<ViewTransition>` in the page and so it would've considered not leaking to the root and so the root animation gets cancelled. But since this is in the root it now doesn't animate this transition. Now we're more pessimistic about that.

By wrapping this `<div>` in `<ViewTransition>` the root no longer needs to animate because the mutation is contained within this ViewTransition.